opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -26,12 +25,19 @@
     private static final Logger log = Logger.getLogger(ServerTracingAutoConfiguration.class.getName());
 
     @Bean
-    public FilterRegistrationBean tracingFilter(Tracer tracer) {
+    @ConditionalOnMissingBean(WebTracingConfiguration.class)
+    public WebTracingConfiguration tracerAutoConfiguration() {
+        return WebTracingConfiguration.builder()
+                .withSkipPattern(WebTracingConfiguration.DEFAULT_SKIP_PATTERN).build();
+    }
+
+    @Bean
+    public FilterRegistrationBean tracingFilter(Tracer tracer, WebTracingConfiguration tracingConfiguration) {
         log.info("Creating " + FilterRegistrationBean.class.getSimpleName() + " bean with " +
-                TracingFilter.class + " mapped to " + "/*");
+                TracingFilter.class + " mapped to " + "/*, skip pattern is " + tracingConfiguration.skipPattern);
```

Comment:
`tracingConfiguration` won't be `null` and string concat with null works:
http://stackoverflow.com/questions/4260723/concatenating-null-strings-in-java
https://docs.oracle.com/javase/specs/jls/se8/html/jls-5.html#jls-5.1.11